buildbot/buildbot/templates/base.html

```diff
@@ -11,7 +11,8 @@
         {% block head %}{% endblock head %}
     </head>
     <body>
-        <h1>Seashell Buildbot</title>
+        <h1>Buildbot</title>
+        <hr/>
```

Comment:
This hr looks a tiny bit clunky to me… I think I'd rather just put a border-bottom on the h1. I can fiddle with that though.

Reply:
Fixed.